### PR TITLE
Remove redundant partitioning tests TINC

### PR DESCRIPTION
**Pull request by hlinnaka**
I will squash these when pushing, but I keeping them separate for now might help in reviewing.

### CODE REVIEW

**Comment by xinzweb** [comment]
+1. That's a lot of dups.

File: src/test/regress/expected/partition.out [hunk 1]
```diff
@@ -3768,7 +3768,17 @@ where tablename = 'mpp5941';
 (6 rows)
 
 drop table mpp5941;
--- MPP-5984
+-- MPP-5984 - NULL is not allowed in RANGE partition
```
Comment (asubramanya) [inline src/test/regress/expected/partition.out]: I think you need to update the `partition_optimizer.out` file too.

**Comment by hlinnaka** [comment]
On 02/24/2017 12:20 AM, Abhijit Subramanya wrote:
> I think you need to update the `partition_optimizer.out` file too.

Thanks, good catch. Updated.

- Heikki

**Comment by hlinnaka** [comment]
Pushed, thanks!

**Pull request closed without merging**